cmake/ConfigureCompiler.cmake

```diff
@@ -253,7 +253,7 @@ else()
       set(USE_AVX512F OFF)
     endif()
   endif()
-  set(USE_AVX OFF CACHE BOOL "Enable AVX instructions")
+  set(USE_AVX ON CACHE BOOL "Enable AVX instructions")
```

Comment:
That does not seem related to the cherry-pick. Is it necessary?

Reply:
ack